boards/arm/stm32h7b3i_dk/stm32h7b3i_dk.dts

```diff
@@ -50,7 +51,7 @@
 		device_type = "memory";
 		reg = <0xd0000000 DT_SIZE_M(16)>;
 		zephyr,memory-region = "SDRAM2";
-		zephyr,memory-attr = "RAM";
+		zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM) )>;
```

Comment:
Any reason to add parenthesis ?

Following would be lighter:
```suggestion
		zephyr,memory-attr = <DT_MEM_ARM(ATTR_MPU_RAM)>;
```

Reply:
That's actually required otherwise otherwise DTC will complain with: `parse error: expected number or parenthesized expression`